GunShipAlpha/src/Tests.java

```diff
@@ -0,0 +1,41 @@
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+import java.awt.*;
+import java.awt.geom.Point2D;
+
+class GameStateTest {
+
+    @Test
+    void testBehemothPlayerCollision() {
+        GameState gameState = new GameState();
+        Player player = new Player();
+        Behemoth behemoth = new Behemoth(350, 350, 50, 50);
+        gameState.addBehemoth(behemoth);
+        behemoth.moveTowardTarget();
+        boolean isCollision = gameState.checkBehemothPlayerCollisions(player, 800, 600);
+        assertTrue(isCollision);
```

Comment:
i don't see any tests that assertFalse collision - seems like you're missing some cases. 

Also, with all these hard coded values, you should at least put in a comment explaining what the arrangement represents or something.

Reply:
wdym by assert false collisions? like test a scenario that shouldn't be a collision?

Reply:
yes, exactly. different tests should (try to) cover every path the code can take, not just the "happy path" or most obvious case.